scripts/all-tests.sh

```diff
@@ -98,14 +98,12 @@ END
     "unit")
         case "$SHARD" in
             "lms")
-                paver test_system -s lms --extra_args="--with-flaky" || { EXIT=1; }
-                paver coverage
+                SHARD=1 paver test_system -s lms --extra_args="--with-flaky" --cov_args="-p" || { EXIT=1; }
```

Comment:
This was basically so that the directory structure would end up being `reports/lms/shard_1` instead of `reports/lms/shard_lms` (or worse, below would be `reports/common/shard_cms-js-commonlib`).

To add another shard, we could do something like:
```
"lms-1")
     SHARD=1 paver test_system ...
     ;;
"lms-2")
     SHARD=2 paver test_system ...
     ;;
```